src/app/features/auth/services/auth/auth.service.ts

```diff
@@ -0,0 +1,71 @@
+import { inject, Injectable, signal } from '@angular/core';
+import { LocalStorageService } from '@core/services/local-storage/local-storage.service';
+import { Router } from '@angular/router';
+import { catchError, EMPTY, tap } from 'rxjs';
+import { UserResponse } from '@features/auth/interfaces/user-response.interface';
+import { LocalStorageKey } from '@shared/enums/local-storage-key.enum';
+import { AlertService } from '@core/services/alert/alert.service';
+import { AuthApiService } from '../auth-api/auth-api.service';
+import { UserRequest } from '../../interfaces/user-request.interface';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class AuthService {
+  private readonly router = inject(Router);
+
+  private readonly alert = inject(AlertService);
+
+  private authApiService = inject(AuthApiService);
+
+  private localStorage = inject(LocalStorageService);
+
+  public isLoggedIn = signal(true);
+
+  public isAdminIn = signal(false);
```

Comment:
I've already said that we should have auth state as `UserInfo | null` and `isAdmin` should be computed from it.

Reply:
while this is a stub, the task with this will be later, we will fix it there